.scala-steward.conf

```diff
@@ -5,6 +5,5 @@ pullRequests.frequency = "14 days"
 updates.ignore = [
   # as per discussion on sbt/zinc#1236, this is
   # "if it ain't broke don't fix it" territory
-  { groupId = "com.google.protobuf" },
```

Comment:
If you do this, could you pin protobuf-java to 3.x since the latest is 4.x plz, which is not bincompat?
https://repo1.maven.org/maven2/com/google/protobuf/protobuf-java/